library/Fission/CLI/Command/Whoami.hs

```diff
@@ -0,0 +1,48 @@
+-- | Whoami command
+module Fission.CLI.Command.Whoami (command, whoami) where
+
+import           RIO
+import           RIO.ByteString
+
+import           Options.Applicative.Simple (addCommand)
+import           Servant
+
+import qualified Data.ByteString.Char8 as BS
+import qualified System.Console.ANSI as ANSI
+
+import           Fission.Internal.Constraint
+import qualified Fission.Internal.UTF8 as UTF8
+
+import qualified Fission.CLI.Auth as Auth
+import           Fission.CLI.Config.Types
+
+
+-- | The command to attach to the CLI tree
+command :: MonadIO m
+        => HasLogFunc cfg
+        => cfg
+        -> CommandM (m ())
```

Comment:
Not for this PR, but FYI I'm going to write a helper:

```haskell
type MonadCmd cmd = forall m . MonadIO m => CommandM (m cmd)
```

so then we can write:

```haskell
command :: HasLogFunc cfg => cfg -> MonadCmd ()
```

Or something with better naming